frontend/src/logger/ILogger.ts

```diff
@@ -9,7 +9,11 @@ export interface ILogger {
 
   debug(message: string, extra?: Record<string, unknown>, source?: string): void
 
-  error(message: string, extra?: Record<string, unknown>, source?: string): void
+  error(
+    message: Error | string,
+    extra?: Record<string, unknown>,
+    source?: string,
+  ): void
 
   critical(
```

Comment:
question: shall we pass errors to `critical` as well?

Reply:
makes sense to me